public/app/plugins/datasource/cloudwatch/datasourcecfg.cue

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 Grafana Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grafanaplugin
+
+import (
+	common "github.com/grafana/grafana/packages/grafana-schema/src/common"
+	"github.com/grafana/grafana/pkg/plugins/pfs"
+)
+
+// This file (with its sibling .cue files) implements pfs.GrafanaPlugin
+pfs.GrafanaPlugin
+
+composableKinds: DataSourceCfg: {
+	maturity: "merged"
+
+	lineage: {
+		seqs: [
+			{
+				schemas: [
+					{
+						#LogGroup: {
+							// ARN of the log group
+							arn: string
+							// Name of the log group
+							name: string
+							// AccountId of the log group
+							accountId?: string
+							// Label of the log group
+							accountLabel?: string
+						} @cuetsy(kind="interface")
+
+						#AwsAuthType: "keys" |
+							"credentials" |
+							"default" |
+							"ec2_iam_role" |
+							// @deprecated use default
+							"arn" @cuetsy(kind="enum", memberNames="Keys|Credentials|Default|EC2IAMRole|ARN")
```

Comment:
Question: how can I mark "arn" as a deprecated member? Here's the original type https://github.com/grafana/grafana-aws-sdk-react/blob/60f4c81d71a7290f29d05c12c8398991bba3863e/src/types.ts#L8-L11

Reply:
Looks like that the comment is completely ignored. We didn't cover this case but I cannot say without do any test, if cue library is detecting it or not.

If you set the comment at the end of the value, does it work?
```cue
"arn" @cuetsy(kind="enum", memberNames="Keys|Credentials|Default|EC2IAMRole|ARN") // @deprecated use default
```

Reply:
no, it still doesn't work. it generates the following TS which doesn't show the deprecation hint for the `ARN` member.

```ts
export enum AwsAuthType {
  ARN = 'arn',
  Credentials = 'credentials',
  Default = 'default',
  EC2IAMRole = 'ec2_iam_role',
  Keys = 'keys',
} // @deprecated use default
```

Reply:
Ok! Let me open an issue to cover this use case in cuetsy.